code/index.html

```diff
@@ -1,30 +1,324 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <!-- Viewport fit: cover is for iphone overflow -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover" />
```

Comment:
I had no idea about viewport-fit=cover, will keep that in mind - very well spotted.

Reply:
Thank you! It took a lot of googling. When I tested on my cellphone, I noticed that the sides cut off in landscape and I wanted it to fit the entire width of the screen!